_data/vendors.yml

```diff
@@ -408,6 +408,15 @@
   pricing_source: https://trello.com/pricing
   updated_at: 2018-10-17
 
+- name: TeamViewer
+  url: https://teamviewer.com
+  base_pricing: €27.90 per month
+  sso_pricing: €833.00 per monh
+  percent_increase: 3085% [^teamviewer]
```

Comment:
This should be 2886%:
`(833-27.90)/27.90*100` is 28.8566

Reply:
true.  it also has a spelling mistake.